docs/application/native/guides/security/security-tip.md

```diff
@@ -70,6 +70,8 @@ Because these privileges are used to access very sensitive user data, when an ap
 ## Communicating Securely Between Applications
 
 Tizen supports APIs for communicating between application processes. You can use various methods when developing your Tizen application to ensure secure communication: file sharing, message ports, and data control.
+> **Note**  
+> In general, the best approach for user data security is to minimize the use of APIs that access sensitive or personal user data. Be careful when communicating between applications. If you want transmit the user data, consider if there is a way that your application logic can be implemented using a hash or non-reversible form of the data.
```

Comment:
@kilig Kindly update the note tag according to the latest style guide. For more information: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md

Request you to kindly modify the content as it is not aligning with our language tone/pitch. Also, does this sentence means the following:

\> [!NOTE]
\> In general, the best approach for user data security is to carefully select those APIs that access sensitive or personal data. For example, if you want to transmit the data, consider if there is a way that your application logic can be implemented using a hash or non-reversible form of the data.

Please feel free to correct me for the technical meaning.